crates/kubelet/src/server.rs

```diff
@@ -20,7 +26,14 @@ pub async fn start_webserver<T: 'static + Provider + Send + Sync>(
     provider: Arc<Mutex<T>>,
     address: &SocketAddr,
 ) -> Result<(), failure::Error> {
-    let service = make_service_fn(move |_conn: &AddrStream| {
+    let identity = tokio::fs::read("/home/rylevick/krustlet/certificate.pfx")
```

Comment:
Did we want to use a pfx bundle or a separate cert and key combo?

Reply:
Does OpenSSL's CLI allow you to generate pfx bundles?

Reply:
This is what I did locally on my Ubuntu box:
```
openssl genrsa 2048 > host.key
chmod 400 host.key
openssl req -new -x509 -nodes -sha256 -days 365 -key host.key -out host.cert
openssl pkcs12 -export -out certificate.pfx -inkey host.key -in host.cert
```